documentation/source/user_section/tutorials/lesion-analysis/lesion-statistics/lesion-metrics.rst

```diff
@@ -0,0 +1,60 @@
+Compute lesion morphometric measures using ``sct_analyze_lesion``
+#################################################################
+
+The ``sct_analyze_lesion`` function computes various morphometric measures on segmented lesions and spinal cord.
+In case of multiple lesions, the function assigns an ID value to each lesion (1, 2, 3, etc.) and then outputs morphometric measures for each lesion to an XLS file.
```

Comment:
```suggestion
Here, we will see how to compute various morphometric measures on segmented lesions. For example, the number of lesions, their size, etc. The statistics will be output in an Excel (XLS) file.

In case of multiple lesions, the function assigns an ID value to each lesion (1, 2, 3, etc.) and then outputs morphometric measures for each lesion to an XLS file.
```